staging/vhost-device-gpu/src/virt_gpu.rs

```diff
@@ -829,3 +830,68 @@ impl VirtioGpuBackend for VirtioGpu {
         Ok(OkNoData)
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use std::sync::{Arc, Mutex};
+
+    use super::{VirtioGpu, VirtioGpuBackend, VirtioGpuResource, VirtioGpuRing, VirtioShmRegion};
+    use rutabaga_gfx::{
+        ResourceCreateBlob, RutabagaBuilder, RutabagaComponentType, RutabagaHandler,
+    };
+    use vhost_user_backend::{VringRwLock, VringT};
+    use vm_memory::{GuestAddress, GuestMemoryAtomic, GuestMemoryMmap};
+
+    fn new_2d() -> VirtioGpu {
+        let rutabaga = RutabagaBuilder::new(RutabagaComponentType::Rutabaga2D, 0)
+            .build(RutabagaHandler::new(|_| {}), None)
+            .unwrap();
+        VirtioGpu {
+            rutabaga,
+            resources: Default::default(),
+            fence_state: Arc::new(Mutex::new(Default::default())),
+        }
+    }
+
+    #[test]
+    fn test_gpu_backend_success() {
+        let mem = GuestMemoryAtomic::new(
+            GuestMemoryMmap::<()>::from_ranges(&[(GuestAddress(0), 0x1000)]).unwrap(),
+        );
+        let vring = VringRwLock::new(mem.clone(), 0x1000).unwrap();
+        vring.set_queue_info(0x100, 0x200, 0x300).unwrap();
+        vring.set_queue_ready(true);
```

Comment:
This test doesn't test any components in this file, it kind of tests the rutabaga crate only. The variables `mem` and `vring` are not used after they are setup here.  I don't think we need this test at all.